docs/Streams.md

```diff
@@ -33,8 +33,16 @@ For peer initiated streams, the app gets a `QUIC_CONNECTION_EVENT_PEER_STREAM_ST
 # Sending
 
 An app can send on any locally initiated stream or a peer initiated bidirectional stream. The app uses the [StreamSend](api/StreamSend.md) API to send data.
-MsQuic holds on to any buffers queued via [StreamSend](api/StreamSend.md) until they have been completed via the `QUIC_STREAM_EVENT_SEND_COMPLETE` event:
-the app must not free, reuse or otherwise access the buffers provided through [StreamSend](api/StreamSend.md) before the matching completion notification.
+
+MsQuic takes ownership of any buffers successfully queued via [StreamSend](api/StreamSend.md). Buffer ownership is
+returned to the application via the `QUIC_STREAM_EVENT_SEND_COMPLETE` event. The application must not free, reuse or
+otherwise access a buffer provided to [StreamSend](api/StreamSend.md) until the matching
+`QUIC_STREAM_EVENT_SEND_COMPLETE` event.
+
+![Note]
+> `QUIC_STREAM_EVENT_SEND_COMPLETE` does not mean the data has been received by the peer application layer.
+> It only means that MsQuic no longer needs the app send buffer and give the owernship back to the application. The app
+> should assume the data has been successfully transmitted based on this notification.
```

Comment:
"data has been successfully **transmitted** based on this notification" -would this statement cause confusions as it could be just buffered by msquic or afd?

Reply:
Yeah, let's go ahead and remove this last sentence.

Reply:
There is a "not" missing. I meant to say "The app should *not* assume the data has been successfully transmitted".

I think there is value in clarifying it, we got some questions about it a month ago or so.

Reply:
```suggestion
> should *not* assume the data has been successfully transmitted based on this notification.
```